src/test/java/uk/gov/ons/census/casesvc/service/RefusalProcessorTest.java

```diff
@@ -71,10 +70,10 @@ public void shouldProcessARefusalReceivedMessageSuccessfully() {
             EventType.CASE_UPDATED,
             expectedRefusal,
             managementEvent.getEvent(),
-            expectedRefusal.getResponseDateTime());
+            OffsetDateTime.parse(expectedRefusal.getResponseDateTime()));
   }
 
-  @Test(expected = RuntimeException.class)
```

Comment:
Without re-throwing and expecting the exception here this test will pass even if the exception is not thrown

Reply:
not sure what you mean here @AdamHawtin, test fails unless RuntimeException?

Reply:
If a runtime exception is thrown and the message doesn't match, it'll fail but if not exception is thrown the test will still pass.
You can comment out `underTest.processRefusal(managementEvent);` and the test passes.